src/crud.js

```diff
@@ -0,0 +1,148 @@
+import dots from '../assets/images/dots.png';
+import deleteIcon from '../assets/images/delete.png';
+import saveIcon from '../assets/images/save.jpeg';
+
+class ToDo {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const deleteTodos = (e) => {
+  const btnClass = e.target.className;
+  const btnId = btnClass.split('-');
+  const id = parseInt(btnId[1], 10);
+  let existingTodos = JSON.parse(localStorage.getItem('todos'));
+  existingTodos = existingTodos.filter((todos, index) => index !== id);
+  e.target.parentNode.remove();
+  // eslint-disable-next-line no-return-assign
+  existingTodos.forEach((task, i) => task.index = i + 1);
+  localStorage.setItem('todos', JSON.stringify(existingTodos));
+  // eslint-disable-next-line no-use-before-define
+  showToDos();
+};
+
+const saveTodos = (e) => {
+  const saveBtn = e.target;
+  const existingTodos = JSON.parse(localStorage.getItem('todos'));
+  const btnClass = saveBtn.className;
+  const btnId = btnClass.split('-');
+  const id = parseInt(btnId[1], 10);
+  const taskList = document.querySelector(`#tasks-${id}`);
+  const saveEdit = document.querySelector(`.save-${id}`);
+  const deleteEdit = document.querySelector(`.delete-${id}`);
+  const inputId = document.querySelector(`#activity-${id}`);
+  const editBtn = document.getElementById(`${id}`);
+  editBtn.style.display = 'block';
+
+  existingTodos[id].description = inputId.value;
+  localStorage.setItem('todos', JSON.stringify(existingTodos));
```

Comment:
[OPTIONAL]

- I can see that you used local storage multiple times. I would kindly suggest you write separate functions that will add and get items from localStorage so as to make your code reusable and prevent repeating yourself 😄 .

Reply:
Thank you @vickymarz